docs/cluster_manifest.md

```diff
@@ -0,0 +1,215 @@
+Postgres Cluster Manifest
+=========================
+
+Individual postgres clusters are described by the Kubernetes *cluster manifest*
+that has the structure defined by the `postgres CRD` (custom resource
```

Comment:
is it worth linking to the actual [source code](https://github.com/zalando-incubator/postgres-operator/blob/922b3fa40852db2c984dca4c6c75285503fba650/pkg/spec/postgresql.go#L98) ?

Reply:
No. The audience of the documentation does not necessary know what to make of it.